src/main/scala/com/gu/kindlegen/app/Lambda.scala

```
@@ -122,7 +122,7 @@ class Lambda(settings: Settings, date: LocalDate) extends Logging {
 
     val deletePublicDir = publisher
       .undirect(settings.s3.publicDirectory)
-      .andThen { case Failure(error) => logger.warn("Deleting the public directory failed!", error) }
+      .andThen { case Failure(error) => logger.error("Deleting the public directory failed!", error) }
```

Side fix: since the future and the subsequent futures will fail, we should log an error. It should be changed back to `warn` if the error is ignored.